internal/cloud/errors.go

```diff
@@ -58,3 +58,46 @@ func incompatibleWorkspaceTerraformVersion(message string, ignoreVersionConflict
 	description := strings.TrimSpace(fmt.Sprintf("%s\n\n%s", message, suggestion))
 	return tfdiags.Sourceless(severity, "Incompatible Terraform version", description)
 }
+
+type multiErrors []error
+
+// Append errs to e only if the individual error in errs is not nil
+//
+// If any of the errs is itself multiErrors, each individual error in errs is appended.
+func (e *multiErrors) Append(errs ...error) {
+	for _, err := range errs {
+		if err == nil {
+			continue
+		}
+		if errs, ok := err.(multiErrors); ok {
+			*e = append(*e, errs...)
+		}
+		*e = append(*e, err)
```

Comment:
The comment above says, `// If any of the errs is itself multiErrors, each individual error in errs is appended.`, however, it looks like the bulk multiErrors err is also appended to e even after appending the individual error in line#73, is this intentional?

Reply:
fixed